src/rt/minfo.d

```diff
@@ -339,8 +353,7 @@ struct ModuleGroup
                 else
                 {
                     auto midx = edges[sp.curMod][sp.curDep];
-                    // if midx is -1, then this isn't part of this DSO.
-                    if (midx != -1 && !bts(reachable, midx))
+                    if (!bts(reachable, midx))
```

Comment:
Seems correct but slightly unrelated (though you prolly double checked for the old implementation).
That means we could change the type of edges to `uint`, right?